examples/supertuxkart/main.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	sdk "agones.dev/agones/sdks/go"
+)
+
+// logLocation is the path to the location of the Super Tux Kart log file
+const logLocation = "/.config/supertuxkart/config-0.10/server_config.log"
+
+// main intercepts the log file of the Super Tux Kart gameserver and uses it
+// to determine if the game server is ready or not.
+func main() {
+	input := flag.String("i", "", "path to server_linux.sh")
```

Comment:
This comment is now out of date. The flag is the command + arguments to execute the stk server binary.